specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -5679,6 +5693,14 @@
           "description": "The sku description of the Azure plan for the subscription.",
           "type": "string",
           "readOnly": true
+        },
+        "suspensionReasons": {
+          "description": "The suspension reason for a subscription. Applies only to legacy subscriptions",
```

Comment:
Instead of "legacy subscriptions", maybe say "Applies only to subscriptions in Microsoft Online Services Program (MOSP) billing accounts." or something like that.

Reply:
Changed.